habitat/core/vector_env.py

```diff
@@ -302,7 +285,7 @@ def _spawn_workers(
     def current_episodes(self):
         self._is_waiting = True
         for write_fn in self._connection_write_fns:
-            write_fn((EPISODE_COMMAND, None))
+            write_fn((CALL_COMMAND, ("current_episode", None)))
```

Comment:
Maybe, worth to keep consts `EPISODE_COMMAND` otherwise we have string const that are related to code function names (reflection) buried in the code.

```suggestion
            write_fn((CALL_COMMAND, (EPISODE_COMMAND, None)))
```

Reply:
Done